### ISSUE TITLE
Mask returned in serverless automatic annotation does not agree with external result

### ISSUE THREAD

**Issue by hermda02** [cvat-ai/cvat#7943]
### Actions before raising this issue

- [X] I searched the existing issues and did not find anything similar.
- [X] I read/searched [the docs](https://docs.cvat.ai/docs/)

### Steps to Reproduce

1. Create a simple serverless `detector` function which produces a binary mask.
2. Return an inference result with the following format:         
`results.append({
            "type": "mask",
            "confidence": str(prob),
            "label": "Net",
            "mask": cvat_mask,   
        })
`
where cvat_mask = binary_mask + [bounding box corners]
3. Select "Automatic annotation" on an image within a CVAT task.

### Expected Behavior

Expect CVAT to return an annotated image with the input binary mask applied, like the attached image. (These are the same inference results that go into CVAT).
![image_prediction](https://github.com/cvat-ai/cvat/assets/17911822/f936cfaa-ffa8-4daf-a8d5-19d2cbf2326d)

### Possible Solution

Documentation concerning the format required by CVAT in order for the binary mask to be properly handled.

### Context

I've created a serverless function with my own pre-trained UNet model which returns masks for a given image by using PyTorch. When running inference on both my own machine and using Nuclio I am able to retrieve the same results as far as model outputs and mask details go.

However, when sending the mask in to CVAT, it returns a messy image:
![Screenshot from 2024-05-24 12-22-05](https://github.com/cvat-ai/cvat/assets/17911822/980bc470-8f97-4b70-9775-e93eb5eff251)

This appears to be an issue with going from a binary mask to the RLE solution (https://github.com/cvat-ai/cvat/blob/develop/cvat/apps/lambda_manager/views.py, lines 738-743).  

### Environment

```Markdown
Server version: 2.9.0
Core version: 12.0.0
Canvas version: 2.18.0
UI version: 1.58.0
```

**Comment by hermda02** [comment]
Unfortunately I was not. If possible, I would like to change the label to a documentation issue.

**Comment by hermda02** [comment]
This seems to be related to #6332 . Any chance you know in which piece of code the mask is parsed and applied to the input image? 

I'm finding that an input block mask (`mask[:500,:500] = 1`) returns a masked image that appears to suffer some type of anti-aliasing:
![image](https://github.com/cvat-ai/cvat/assets/17911822/d1b44a4a-dbfe-47be-8b50-0070b0125e1f)

**Comment by bsekachev** [comment]
Be sure you are sending correct borders.

[0, 0, 500, 500] - will be incorrect in this case
[0, 0, 499, 499] is correct

**Comment by hermda02** [comment]
The borders are correct and are determined by `torchvision.ops.mask_to_boxes`.

Docker log output shows:
```
Borders: [0, 0, 499, 499]
Mask: tensor([[1, 1, 1,  ..., 0, 0, 0],
        [1, 1, 1,  ..., 0, 0, 0],
        [1, 1, 1,  ..., 0, 0, 0],
        ...,
        [0, 0, 0,  ..., 0, 0, 0],
        [0, 0, 0,  ..., 0, 0, 0],
        [0, 0, 0,  ..., 0, 0, 0]])
Sum(mask): tensor(250000)
```

**Comment by hermda02** [comment]
Solution was found by utilizing the `to_cvat_mask` function found here:
https://github.com/cvat-ai/cvat/blob/develop/serverless/openvino/base/shared.py
and implemented here:
https://github.com/cvat-ai/cvat/blob/develop/serverless/openvino/omz/intel/semantic-segmentation-adas-0001/nuclio/model_handler.py
